oshop-dashboard/.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
this should sit in it's own PR called barette/prettier. We want to make PRs as small as possible. Each PR should do one thing and one thing only